toponetx/classes/cell_complex.py

```diff
@@ -1429,6 +1433,18 @@ def all_cell_to_node_codjacnecy_matrix(
         scipy.sparse.csr.csc_matrix | tuple[dict, dict, scipy.sparse.csc_matrix]
             The adjacency matrix, if `index` is False, otherwise
             index of cells, adjacency matrix, if 'index' is True
+
+        Note
+        ----
+        Two cells (1 dimensional or 2 dimensional) are s-coadjacent iff
+        they share a vertex.
+
+        Example
```

Comment:
NumPy's documentation style dictates that this headline should read Examples even if there is only one (same throughout this file).

Reply:
I am not sure I understand this comment.